docs/content/en/docs/getting-started/optional/irsa.md

```diff
@@ -47,9 +47,24 @@ You must use a single OIDC provider per EKS Anywhere cluster, which is the best
 
 1. Upload the `discovery.json` file to the S3 bucket:
     ```bash
-    aws s3 cp --acl public-read ./discovery.json s3://$S3_BUCKET/.well-known/openid-configuration
+    aws s3 cp ./discovery.json s3://$S3_BUCKET/.well-known/openid-configuration
     ```
 
+1. Use a [bucket policy](https://docs.aws.amazon.com/AmazonS3/latest/userguide/add-bucket-policy.html) to grant public read access to the `discovery.json` file:
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": "*",
+      "Action": "s3:GetObject",
+      "Resource": "arn:aws:s3:::$S3_BUCKET/.well-known/openid-configuration"
+    }
+  ]
+}
+```
```

Comment:
Let's just document the bucket policy once. I would say keep the policy on line 150 and get rid of it here